docusaurus/docs/test-a-plugin/guides/feature-toggles.md

```diff
@@ -0,0 +1,85 @@
+---
+id: feature-toggles
+title: Feature toggles
+description: How to use feature toggles in e2e tests
+draft: true
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - testing
+  - e2e
+  - feature toggles
+sidebar_position: 1
+---
+
+# Introduction
+
+Feature toggles are used to enable or disable features in Grafana. Feature toggles can have big impact on how a plugin behaves, so you may need to cater for that within you e2e tests. `@grafana/plugin-e2e` have a few fixtures that helps while working with feature toggles.
+
+## Passing feature toggles to Grafana
+
+The easisest way to configure feature toggles that are available across the entire Grafana stack is by specifying [environment variables](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#override-configuration-with-environment-variables) when starting your Grafana instance.
```

Comment:
This is just an idea and maybe wrong: would it be useful to also share a short snippet on how to enable a certain feature flag using the `GF_` env variables? I feel like it's not that easy to figure out from the other docs. (And with that said, probably it's not the responsibility of this doc to fix that, and maybe it's not good if we have things documented in multiple places. Maybe @josmperez has an opinion on this.)

Reply:
Yeah good question. I thinking referring to the configuration docs is better as this might be subject to change in the future. WDYT @sympatheticmoose?

Reply:
I'd lean towards including an example. But no strong preference.

> this might be subject to change in the future

How do you expect this to change?